packages/xrpc/src/util.ts

```diff
@@ -163,5 +164,8 @@ export function httpResponseBodyParse(
       }
     }
   }
+  if (data instanceof ArrayBuffer) {
+    return new Uint8Array(data)
```

Comment:
snuck this in. we have `Uint8Arrays` in the types but they were actually ArrayBuffers